dynawo/sources/Simulation/DYNSimulation.cpp

```diff
@@ -922,7 +922,14 @@ Simulation::simulate() {
         connectedComponents_ = data_->findConnectedComponents();
       }
     }
+
+    boost::optional<int> iterationStep = jobEntry_->getOutputsEntry()->getCurvesEntry()->getIterationStep();
+    boost::optional<double> timeStep = jobEntry_->getOutputsEntry()->getCurvesEntry()->getTimeStep();
+    if (iterationStep && timeStep) {
+      throw DYNError(DYN::Error::SIMULATION, IterationStepAndTimeStepBothDefined);
```

Comment:
I think this is a check on the consistency of what indicated in the xml, I am not sure it should be done in DYNSimulation but in the API part, if possible. @rosiereflo your opinion?

Reply:
I totally agree, cf my comment on JobXmlHandler.cpp :)